test/05_local_listen.js

```diff
@@ -3,21 +3,18 @@
 
 var LL              = require('../lib/local_listen');
 var RS              = require('../lib/remote_send');
-var Base            = require('../lib//base');
 var C               = require('../lib/common').common();
 var TestLib         = require('./lib/test');
 
-var Dgram           = require("dgram");
 var U               = require('util');
-var Net             = require('net');
 var FS              = require('fs');
 
 
 TestLib.Test( function( test, testlib, config ) {
 
     var _stream_test = function( t, type, port, host ) {
         // The listener
-        var ll = new LL.Stream( type, port, host,
+        new LL.Stream( type, port, host,
```

Comment:
this looks suspicious, as we're passing in ```ll``` in line 19/22

Reply:
No, that's a function arg, which is actually shadowing this name.